react-native/debugging/handle-gradle-dependencies-clash.mo.md

```diff
@@ -0,0 +1,116 @@
+# [MO] Handle gradle dependencies versions clashes
```

Comment:
Maybe change the title to "Handle version clashes between Gradle dependencies"? (I'd also rename "clashes" to "conflicts")

Reply:
👍